extensions/1bookmark/src/handle-signin.ts

```diff
@@ -0,0 +1,40 @@
+import { join } from 'node:path'
+import { hostname } from 'node:os'
+import { getPreferenceValues, showToast, Toast } from '@raycast/api'
+import axios from 'axios'
+
+interface Preferences {
+  apiUrl: string
+}
+
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x

Reply:
I understood and handled it well. b0fdddb2948acc6bb55201c804bd2e611e21b67e